editor/modes/visual-editor/style.scss

```diff
@@ -256,38 +256,25 @@ $sticky-bottom-offset: 20px;
 	}
 }
 
-.editor-visual-editor__placeholder {
+.editor-visual-editor__placeholder[type=text] {
 	margin-left: auto;
 	margin-right: auto;
 	max-width: $visual-editor-max-width;
 	position: relative;
 	padding: $block-padding;
-	border: none;
+	border: 2px solid transparent;
```

Comment:
Minor: Adding the border pushes the text to the right by `2px`. I only noticed it upon refreshing from `master` to this branch, but it also appears slightly offset from both the title and following inserter.

~~We might consider a `margin: 0 -2px` to offset (optionally applying both margin and border only ever to the left side).~~

Can't do margin because we need the `auto`. Since we're already applying `position: relative`, we could do `left: -2px` instead.